client/lib/plans/constants.js

```diff
@@ -47,6 +50,9 @@ export const plansList = {
 		getTitle: () => i18n.translate( 'Free' ),
 		getPriceTitle: () => i18n.translate( 'Free for life' ),
 		getProductId: () => 1,
+		getStoreSlug: () => PLAN_FREE,
+		getPathSlug: () => 'beginner',
```

Comment:
This is actually a working route: 
https://github.com/Automattic/wp-calypso/pull/6207/files#diff-b1008f50a6a54c403179ffebf1da6bc0L48

And in fact this route is broken and throws an error on prod:
https://wordpress.com/checkout/piszek.com/beginner
`Invalid plan product slug: free_plan`

Sooo, this is another small bug this PR fixes.